kibana/tests/kibana_test.py

```diff
@@ -427,6 +427,21 @@ def test_service_annotations():
     ]
     assert s == "0.0.0.0/0"
 
+def test_adding_a_loadBalancerIP():
```

Comment:
Black formatter requires 2 blanck lines between each functions:
```suggestion

def test_adding_a_loadBalancerIP():
```